.github/workflows/release.yml

```diff
@@ -25,15 +25,24 @@ jobs:
       - name: Install dependencies
         run: npm pkg delete scripts.prepare && npm ci
 
+      - uses: radixdlt/public-iac-resuable-artifacts/fetch-secrets@main
+        with:
+          role_name: ${{ secrets.AWS_ROLE_NAME_CONNECTOR_EXTENSION_SECRETS }}
+          app_name: 'connector-extension'
+          step_name: 'connector-extension-webchrome-store-secrets'
+          secret_prefix: 'GH'
+          secret_name: ${{ secrets.AWS_CONNECTOR_EXTENSION_WEBCHROME_STORE_SECRET_ARN }}
+          parse_json: true
+
       - name: Github PreRelease
         if: startsWith(github.ref_name, 'release/') || github.ref == 'refs/heads/develop'
         env:
           VITE_GITHUB_REF_NAME: ${{ github.ref_name }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          GOOGLE_CLIENT_ID: ${{ secrets.CLIENT_ID }}
-          GOOGLE_CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
-          GOOGLE_REFRESH_TOKEN: ${{ secrets.REFRESH_TOKEN }}
-          GOOGLE_EXTENSION_ID: ${{ secrets.EXTENSION_ID }}
+          GOOGLE_CLIENT_ID: ${{ secrets.GH_CLIENT_ID }}
+          GOOGLE_CLIENT_SECRET: ${{ secrets.GH_CLIENT_SECRET }}
+          GOOGLE_REFRESH_TOKEN: ${{ secrets.GH_REFRESH_TOKEN }}
+          GOOGLE_EXTENSION_ID: ${{ secrets.GH_EXTENSION_ID }}
```

Comment:
@alpeto9 you need to change to `env.` because fetched secrets are exposed as env var.